### DISCUSSION TITLE
AS5600 Magnetic Encoder

### DISCUSSION THREAD

**Discussion by DjGuuOmsk** [tttapa/Control-Surface#748] [Q&A]
I apologize if the question is stupid or has already been raised, but is it possible to connect a magnetic encoder? I have already made controllers with regular encoders, but the magnetic encoder is perfect for jog for DJ controllers.
I bought "AS5600 Magnetic Encoder" modules for new controller:
https://www.amazon.com/AS5600-Magnetic-Encoder-Induction-Measurement/dp/B09LVFRHG1/ref=sr_1_1?keywords=as5600&qid=1653230979&sr=8-1

**Comment by tttapa** [comment]
It seems to be a magnetic position sensor, it doesn't output its movement in quadrature encoding.

But as long as you're able to get the position out of it over I²C, I don't see why you couldn't combine it with Control Surface.
